### DISCUSSION TITLE
sPyderChron/New Proposal

### DISCUSSION THREAD

**Discussion by jirhiker** [NMGRL/pychron#2187] [Ideas]
This is a discussion on a new pychron proposal potentially focused on developing a web based data reduction platform for argon geochron/noble gas geochem/isotope geochem

**Comment by jirhiker** [comment]
@nelsiverson  suggested the name sPyderChron

**Reply by jirhiker** [reply]
@nelsiverson looking for a good sPyderChron logo, any suggestions

**Reply by nelsiverson** [reply]
![sPyderChron](https://user-images.githubusercontent.com/33048412/206034609-e7b9dc1f-ec83-4419-afe3-8b49d4e83327.png)

**Reply by nelsiverson** [reply]
Here is a better one using a clipart spider. 

![sPyderChron-2](https://user-images.githubusercontent.com/33048412/206035184-35b08423-13cd-4c57-ad44-423cdaa42598.png)

**Comment by jirhiker** [comment]
There is significant potential in sPyderChron, however there is some hesitation.  sPyderChron has a few easily recognizable benefits; web based - modern technology to build ui, hosted software so its easier to provide more accessibility. 

BUT there are solutions to pychron's current accessibility issues that do not involve rewriting all of pychron's data reduction functionality. Web apps are gaining in popularity however desktop apps still in the game. One potential path forward is to greatly improve/build OS-specific installation wizards.  This could make installing pychron on par with any other python-based GUI application.  The installation package would probably be completely self-contained, e.g. includes its own python interpreter.  

I think sPyderChron is a worthwhile project but it should be spun off of pychron and focus more on simple Ar-Ar calculations and visualizations instead of a full data management platform.  The main reason I say this is because sPyderChron could reach a much greater audience if it was somewhat data collection agnostic.  Essential sPyderChron would need a simple but robust ETL system for ingesting data.  sPyderChron could be connected to pychron data but it also would need the ability to reduce any other data format.
